Makefile

```diff
@@ -17,73 +20,108 @@ SWAGGER_URL     ?= https://github.com/kubernetes/kubernetes/raw/${KUBE_VERSION}/
 OPENAPI_DIR     := provider/pkg/gen/openapi-specs
 OPENAPI_FILE    := ${OPENAPI_DIR}/swagger-${KUBE_VERSION}.json
 SCHEMA_FILE     := provider/cmd/pulumi-resource-kubernetes/schema.json
+
 GOPATH			:= $(shell go env GOPATH)
 
 JAVA_GEN 		 := pulumi-java-gen
 JAVA_GEN_VERSION := v0.5.2
 
 WORKING_DIR     := $(shell pwd)
+CODEGEN_PATH    = bin/${CODEGEN}
+pulumictl := bin/pulumictl
 TESTPARALLELISM := 4
 
-openapi_file::
-	@mkdir -p $(OPENAPI_DIR)
-	test -f $(OPENAPI_FILE) || curl -s -L $(SWAGGER_URL) > $(OPENAPI_FILE)
+# The general form for this Makefile is
+#  - faithfully represent dependencies between build targets as files, where possible (i.e., like a normal Makefile)
+#  - use phony targets to give the CI system a kind of API for building in stages
+
+.PHONY: default
+default: build
```

Comment:
looks like build only includes the sdks - so might want to add the provider for completeness